### PR TITLE
Update to Junit 5

### PR DESCRIPTION
**Pull request by kadir1243**
This pull contains:
* Update to Junit 5

### CODE REVIEW

**Review by NoComment1105 — APPROVED** [review]
LGTM

File: enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java [hunk 1]
```diff
@@ -26,21 +24,18 @@ public void testIntegrity() throws Exception {
             contents = Sets.newHashSet(new String(in.readAllBytes(), StandardCharsets.UTF_8).split("\\R"));
         }
 
-        try (FileSystem fs = Jimfs.newFileSystem()) {
+        Path path = Path.of("src/test/resources/recaf.mappings");
```
Comment (ByMartrixx) [inline enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java:27]: This shouldn't be writing to the input file, it probably should be writing to a temp one

File: enigma/src/test/resources/recaf.mappings [hunk 2]
```diff
@@ -1,10 +1,18 @@
 a testClass
-a.a Z testField
```
Comment (ByMartrixx) [inline enigma/src/test/resources/recaf.mappings:2]: Also, this file shouldn't be modified

File: enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java (hunk 1 above)
Comment (kadir1243) [reply enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java:19]: Now it is not writes to input

File: enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java [hunk 3]
```diff
@@ -1,46 +1,35 @@
...
 public class TestRecaf {
 
     @Test
     public void testIntegrity() throws Exception {
-        Set<String> contents;
-        try (InputStream in = getClass().getResourceAsStream("/recaf.mappings")) {
-            contents = Sets.newHashSet(new String(in.readAllBytes(), StandardCharsets.UTF_8).split("\\R"));
-        }
+        Path path = Path.of("src/test/resources/recaf.mappings");
+        Set<String> contents = new HashSet<>(Files.readAllLines(path));
```
Comment (ByMartrixx) [inline enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java:20]: This should be the same as before, we want to use the file from the same jar as the current class

**Review by ByMartrixx — CHANGES_REQUESTED** [review]

File: enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java (hunk 3 above)
Comment (kadir1243) [reply enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java]: I readded jimfs

**Review by ByMartrixx — APPROVED** [review]

**Comment by ByMartrixx** [comment]
Sorry for the delay, I completely forgot about this pr

**Pull request merged**